app/src/main/java/mod/pranav/dependency/resolver/DependencyResolver.kt

```diff
@@ -127,10 +127,8 @@ class DependencyResolver(
         open fun invalidPackaging(artifact: Artifact) {}
     }
 
-
     fun resolveDependency(callback: DependencyResolverCallback) {
         eventReciever = callback
-        // this is pretty much the same as `Artifact.downloadArtifact()`, but with some modifications for checks and callbacks
```

Comment:
please keep the comments as they are, they make sure future contributors don't make changes that break stuff